napalm/nxos_ssh/nxos_ssh.py

```diff
@@ -453,21 +453,15 @@ def _send_command(self, command, raw_text=False, cmd_verify=True):
         """
         return self.device.send_command(command, cmd_verify=cmd_verify)
 
-    def _send_command_list(self, commands, expect_string=None):
-        """Wrapper for Netmiko's send_command method (for list of commands."""
-        output = ""
-        for command in commands:
-            output += self.device.send_command(
-                command,
-                strip_prompt=False,
-                strip_command=False,
-                expect_string=expect_string,
-            )
-        return output
+    def _send_command_list(self, commands, expect_string=None, **kwargs):
+        """Send a list of commands using Netmiko"""
+        return self.device.send_multiline(
+            commands, expect_string=expect_string, **kwargs
+        )
```

Comment:
Use new Netmiko 4.x way of doing this. Add **kwargs so that read_timeout can be passed in.

Reply:
I need support to understand this....